scripts/upgrade

```diff
@@ -64,8 +64,11 @@ path_url=$(ynh_normalize_url_path $path_url)
 # DOWNLOAD, CHECK AND UNPACK SOURCE
 #=================================================
 
+# Create a temporary directory
+tmpdir="$(ynh_smart_mktemp 10)"
```

Comment:
In `ynh_clean_setup` you should remove this directory

Reply:
Hm OK, so we should test that the `$tmpdir` variable is set, and the corresponding directory exists before removing it. Is there an example already implemented somewhere (lazy mode)...? :sleeping:

Reply:
Implemented in https://github.com/YunoHost-Apps/ttrss_ynh/commit/122ad91d7c727564a1bfb3e46ca18e2c6e13713f
Can you please review it? :wink: